R/rdataretriever.R

```diff
@@ -309,7 +309,15 @@ check_for_retriever = function(...) {
                             '/miniconda3/bin',
                             '/miniconda2/bin')
         for (i in possible_pathes) {
-            Sys.setenv(PATH = paste(Sys.getenv('PATH'), ':', home_dir, i, sep = ''))
+            Sys.setenv(PATH = paste(Sys.getenv('PATH'), ';', home_dir, i, sep = ''))
+        }
+
+        ## Other very likely pathes for Windows users:
```

Comment:
@Pakillo, Thanks for catching this. I have made some tests and I think we could add a few lines

`## Other very likely pathes for Windows users: `> `# paths on Windows if installed using executable`

Since we want this to run on windows only, we could  include a check
```R
more_win_paths = c('C:\\',
                         'C:\\Program files\\',
                         'C:\\Program files (x86)\\')

if (Sys.info()[['sysname']] == "Windows")
  for (i in more_win_paths) {....
```